mathesar_ui/src/components/select/Select.svelte

```diff
@@ -31,16 +42,15 @@
   $: setOptions(options);
 </script>
 
-<Dropdown bind:isOpen contentClass="select {contentClass}" {triggerClass}>
+<Dropdown ariaControls="select-value-{selectId}" {ariaLabel} bind:isOpen contentClass="select {contentClass}" {triggerClass}>
   <svelte:fragment slot="trigger">
     {value?.[labelKey]}
   </svelte:fragment>
   
   <svelte:fragment slot="content">
-    <ul>
+    <ul id="select-value-{selectId}" tabindex="0" role="listbox" aria-expanded="true">
       {#each options as option (option[idKey])}
-        <li class:selected={option[idKey] === value[idKey]}
-            on:click={() => setValue(option)}>
+        <li role='option' class:selected={option === value} on:click={() => setValue(option)}>
```

Comment:
This line seems to be affected during merge.

It should be:
```suggestion
        <li role='option' class:selected={option[idKey] === value[idKey]} on:click={() => setValue(option)}>
```